--- DownloadMoodle.php.backup	2022-12-05 15:30:09
+++ DownloadMoodle.php	2022-12-05 15:30:37
@@ -62,6 +62,7 @@
         }
         // Download example: https://download.moodle.org/download.php/direct/stable310/moodle-latest-310.tgz
         $url =  "https://download.moodle.org/download.php/direct/stable$versioncollapsed/moodle-$exactversion.tgz";
-        run_external_command("wget --continue --timestamping '$url'", "Fetching file failed");
+        $wget = getenv('MOOSH_WGET') ?: 'wget';
+        run_external_command("$wget --continue --timestamping '$url'", "Fetching file failed");
     }
 }
